src/cli/start.rs

```diff
@@ -38,6 +38,9 @@ pub struct StartCommandArguments {
 	#[arg(value_parser = super::validator::key_valid)]
 	#[arg(hide = true)] // Not currently in use
 	key: Option<String>,
+	#[arg(help = "Namespace to store metrics under")]
+	#[arg(env = "SURREAL_NAMESPACE", short = 'n', long = "namespace")]
+	namespace: Option<String>,
```

Comment:
```suggestion
	#[arg(help = "Namespace to store metrics under")]
	#[arg(env = "SURREAL_METRICS_NAMESPACE", short = 'n', long = "namespace")]
	metrics_namespace: Option<String>,
```
The name for this option seems a bit too generic. Should we rename it to something like `metrics_namespace` and `SURREAL_METRICS_NAMESPACE`?

Reply:
Probably also in other places of the code